### PR TITLE
Update Falcon role and Collection

### PR DESCRIPTION
**Pull request by redhatrises**
- Clean up Readmes
- Bump verison to 1.0.0 for linux support
- Update role names

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
-# Ansible Collection - crowdstrike.crowdstrike
+# Ansible Collection - crowdstrike.falcon
 
-Documentation for the collection.
+This collection is focused on downloading and installing the CrowdStrike Falcon Sensor.
+It can also uninstall the Falcon Sensor. Other enhancements are possible in the future
+to add more functionality.
```
Comment (shawndwells) [inline README.md:5]:
```suggestion
This collection is focused on downloading, installing, and removing, CrowdStrike's Falcon sensor on Linux platforms. Windows and OSX coming soon.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+
+# Installation
+
+To install the collection from Ansible Galaxy:
+
+```
+ansible-galaxy collection install crowdstrike.falcon
+```
+
+# Contributing
```
Comment (shawndwells) [inline README.md:15]: Can we also give an example of how to use the role after downloading?

Comment (redhatrises) [reply README.md]: @shawndwells done

File: galaxy.yml [hunk 3]
```diff
@@ -26,8 +26,8 @@ description: >
 
 # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
 # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
-license:
-- Unlicense
+#license:
+#- Unlicense
```
Comment (shawndwells) [inline galaxy.yml:30]: Isn't this needed?

Comment (redhatrises) [reply galaxy.yml:30]: @shawndwells either we specify it as `license` or `license_file`.... because it is specified as `license_files`, it is not needed.

**Review by shawndwells — APPROVED** [review]

**Comment by shawndwells** [comment]
fails linting, ack otherwise

**Comment by redhatrises** [comment]
> fails linting, ack otherwise

Because we have yet to really configure it for collections... also looks like the python code itself is failing.

**Pull request merged**